usaspending_api/etl/transaction_loaders/tests/integration/test_fpds_loader.py

```diff
@@ -48,36 +55,59 @@ def test_load_ids_from_broker(self):
-        # assert load_objects_pre_transaction["transaction_normalized"]["funding_agency_id"] == 1
-        # assert load_objects_pre_transaction["transaction_normalized"]["awarding_agency_id"] == 1
-        # assert 2001 <= load_objects_pre_transaction["transaction_normalized"]["fiscal_year"] <= 2019
+
+        # Run core logic to be tested
+        call_command("load_fpds_transactions", "--ids", *broker_id_list)
+
+        # Assert results
+
+        # Lineage should trace back to the broker records
+        usaspending_transactions = TransactionFPDS.objects.all()
+        assert len(usaspending_transactions) == 3
+        tx_fpds_broker_ref_ids = [_.detached_award_procurement_id for _ in usaspending_transactions]
+        assert 101 in tx_fpds_broker_ref_ids
+        assert 201 in tx_fpds_broker_ref_ids
+        assert 301 in tx_fpds_broker_ref_ids
+        tx_fpds_broker_ref_id_strings = [_.detached_award_proc_unique for _ in usaspending_transactions]
+        assert "101" in tx_fpds_broker_ref_id_strings
+        assert "201" in tx_fpds_broker_ref_id_strings
+        assert "301" in tx_fpds_broker_ref_id_strings
+        tx_norm_broker_ref_id_strings = [_.transaction.transaction_unique_id for _ in usaspending_transactions]
+        assert "101" in tx_norm_broker_ref_id_strings
+        assert "201" in tx_norm_broker_ref_id_strings
+        assert "301" in tx_norm_broker_ref_id_strings
+
+        # All 3 should be under the same award
+        # 1 award created
+        usaspending_awards = Award.objects.all()
+        assert len(usaspending_awards) == 1
+        new_award = usaspending_awards[0]
+
+        # All transactions reference that award and its id
+        tx_norm_awd = [_.transaction.award for _ in usaspending_transactions]
+        assert usaspending_awards[0] == tx_norm_awd[0] == tx_norm_awd[1] == tx_norm_awd[2]
+        tx_norm_awd_ids = [_.transaction.award_id for _ in usaspending_transactions]
+        assert new_award.id == tx_norm_awd_ids[0] == tx_norm_awd_ids[1] == tx_norm_awd_ids[2]
+
+        # And the single award created should refer to the first transaction processed that spawned the award (101)s
+        # TODO: Check if this is true??
+        assert new_award.transaction_unique_id == "101"
+
+        # Award's latest_transaction should point to transaction with latest action_date
+        assert new_award.latest_transaction.transaction_unique_id == "301"
+
+        # Award's earliest_transaction should point to transaction with earliest action_date
+        assert new_award.earliest_transaction.transaction_unique_id == "101"
+
+        # Given action_date additions, the 3rd one should push into 2011
+        assert [2010, 2010, 2011] == [_.transaction.fiscal_year for _ in usaspending_transactions]
 
 
 def _assemble_dummy_broker_data():
+    """
+    Simple way to reuse the field mappings as a def of all required fields, and then use that as teh basis for
```

Comment:
```suggestion
    Simple way to reuse the field mappings as a def of all required fields, and then use that as the basis for
```
You _wish_ you were deadpool